platform/stellar/transaction.go

```diff
@@ -19,42 +17,17 @@ func (p *Platform) GetTxsByAddress(address string) (blockatlas.TxPage, error) {
 }
 
 func (p *Platform) NormalizePayments(payments []Payment) (txs []blockatlas.Tx) {
-	var (
-		wg      sync.WaitGroup
-		txsChan = make(chan blockatlas.Tx, len(payments))
-	)
-
 	for _, payment := range payments {
-		wg.Add(1)
-		go func(pay Payment) {
-			defer wg.Done()
-
-			txHash, err := p.client.GetTxHash(pay.TransactionHash)
-			if err != nil {
-				return
-			}
-
-			tx, ok := Normalize(&pay, p.CoinIndex, txHash)
-			if !ok {
-				return
-			}
-
-			txsChan <- tx
-
-		}(payment)
-	}
-	wg.Wait()
-	close(txsChan)
-
-	for tx := range txsChan {
-		txs = append(txs, tx)
+		if tx, ok := Normalize(&payment, p.CoinIndex); ok {
+			txs = append(txs, tx)
+		}
 	}
 
-	return
+	return txs
 }
 
 // Normalize converts a Stellar-based transaction into the generic model
-func Normalize(payment *Payment, nativeCoinIndex uint, hash TxHash) (tx blockatlas.Tx, ok bool) {
+func Normalize(payment *Payment, nativeCoinIndex uint) (tx blockatlas.Tx, ok bool) {
```

Comment:
Same as at prev comment
(tx blockatlas.Tx, ok bool) => (blockatlas.Tx, bool) ?

Reply:
I don't think this works, as developer won't know what is `bool`, in that case it's preferred to define as `ok bool`

Reply:
it is just styling of functions, there is no difference for the developer

Reply:
Also Normalize => normalize